airbyte-integrations/connectors/source-open-exchange-rates/acceptance-test-config.yml

```diff
@@ -0,0 +1,18 @@
+acceptance_tests:
+  spec:
+    tests:
+      - spec_path: "source_open_exchange_rates/spec.yaml"
+  connection:
+    tests:
+      - config_path: "secrets/config.json"
+        status: "succeed"
+      - config_path: "integration_tests/invalid_config.json"
+        status: "failed"
+  discovery:
+    tests:
+      - config_path: "secrets/config.json"
+  basic_read:
+    tests:
+      - config_path: "secrets/config.json"
+        configured_catalog_path: "integration_tests/configured_catalog.json"
+        empty_streams: []
```

Comment:
When the connector properly implements the `incremental` functionality, you should re-add the `incremental` acceptance tests here.

See [the acceptance tests doc](https://docs.airbyte.com/connector-development/tutorials/build-a-connector-the-hard-way#4-test-the-connector) for more information

Reply:
@mrhallak, could add the `incremental` acceptance tests to this file?

This should do the job:
```yaml
  incremental:
    tests:
      - config_path: "secrets/config.json"
        configured_catalog_path: "integration_tests/configured_catalog.json"
        future_state:
          future_state_path: "integration_tests/abnormal_state.json"
```